packages/components/file-uploader-input/src/utils/selectors.js

```diff
@@ -0,0 +1,17 @@
+import { Helpers } from '@tradeshift/elements';
+
+export const classNames = {
+	fileUploadWrapper: 'file-upload-wrapper',
+	dropBox: 'drop',
+	draggableInfo: 'draggable-info',
+	dragOverState: 'dragover',
+	fileUploadButton: 'file-upload-button'
+};
+console.log('Helpers.classNamesToSelector(classNames)', Helpers.classNamesToSelector(classNames));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS2228)